ludwig/api.py

```diff
@@ -65,6 +65,7 @@
 from ludwig.globals import (
     LUDWIG_VERSION,
     MODEL_HYPERPARAMETERS_FILE_NAME,
+    MODEL_WEIGHTS_FILE_NAME,
```

Comment:
@sanjaydasgupta Thank you for this contribution. Overall it looks really good; I just want to clarify some points.  Would it be possible for you to print out the contents of the model_weights directory -- for example, it is not clear to me how `MODEL_WEIGHTS_FILE_NAME` helps in the path construction?  Let us re-examine it.  Thank you.

Reply:
The name `MODEL_WEIGHTS_FILE_NAME` resolves to the string `model_weights`. And in the following line:
> weights_save_path = os.path.join(model_dir, MODEL_WEIGHTS_FILE_NAME, "ludwig_config.json")

The value of `model_dir` is `.../results/<experiment-name>/model`, so the the path to the config file becomes `.../results/<experiment-name>/model/model_weights/ludwig_config.json`.